doc/dune-files.rst

```diff
@@ -801,17 +801,35 @@ Fields supported in ``<settings>`` are:
 ignored_subdirs
 ---------------
 
-The ``ignored_subdirs`` stanza allows to tell Dune to ignore one or
-more sub-directories. The syntax is as follow:
+The ``ignored_subdirs`` stanza allows to tell Dune to ignore one or more
+sub-directories. The syntax is based on dune's predicate language and allows the
+user the following operations:
+
+- Special values. ``:all`` refers to all sub directories.
```

Comment:
Do we still need this? It's basically the same as `*`

Reply:
`:all` is quite a bit more efficient. Is this optimization important to keep?

Reply:
Actually, I think `:all` is not gonna make so much sense if we allow for globs outside the current dir. For example, if we want a recursive glob, we'll have `**`. In this situation, `:all` is really `**`.

Reply:
If it's just an optimisation, then we could effectively optimise all `*` globs, so yh I don't think it's worth keeping `:all`

Reply:
I got rid of it. The optimization won't even buy much in the common case where the value will be `:standard`.